src/generators/vcf_file_generator.py

```diff
@@ -101,7 +102,7 @@ def _consume_data_source(self):
         assembly_chr_variants = defaultdict(lambda : defaultdict(list))
         assembly_species = {}
         for variant in self.variants:
-            assembly = variant['assembly']
+            assembly = 'R6' if variant['assembly'].startswith('R6') else variant['assembly'].replace('.', '').replace('_', '')
```

Comment:
Nit-pick: personally prefer and regular if-else block over these inline conditionals for readability unless very short/no work going on inside the inline conditional.

Reply:
Thanks for the point. I will leave it in for now. The R6 thing will be fixed in the data sometime soon so I will leave it in for now.